securedrop/Makefile

```diff
@@ -0,0 +1,37 @@
+TAG ?= $(shell git rev-parse HEAD)
+IMAGE = securedrop
+
+.PHONY: assets
+assets:
+	sass
+
+.PHONY: images
+images:
+	docker build $(EXTRA_BUILD_ARGS) -t $(IMAGE):$(TAG) .
+	# combine dockerfiles but strip out first COPY for better cache utilization
+	cat Dockerfile* |sed '/#skip-if-test$$/N;/#skip-if-test/d' | docker build $(EXTRA_BUILD_ARGS) -f - -t $(IMAGE)-test:$(TAG) .
+
+.PHONY: test
+test:
+	docker run --privileged \
```

Comment:
I major concerns with the `privileged` container use here. I dont think it should be a blocker for CI, but I think it should be a blocker from local development documentation and recommendation for users. I won't hold up the PR for it now

Reply:
Also agreed: `--privileged` is _only_ OK in CI, we should _not_ recommend developers run privileged containers on their workstations.

Reply:
I agree, as discussed we need a way to increase entropy in CI and will have to instruct folks in other environment (such as Qubes) to increase the rate of entropy collection.